recipes/fermikit/test-data/test.fastq

```diff
@@ -0,0 +1,100 @@
+@phiX174_29_494_0_1_0_0_2:0:0_3:0:0_0
```

Comment:
We can simply not effort functional tests. Space limitations and so on come to my mind. bioperl is known to be broken and there are several issues. But this has nothing to do with tests, this is perl ;) - the perl packages has some good tests. fastx and similar packages were broken because they were created before we started to test packages with busybox. I hope this is fixed gradually from now on.
Small functional tests are ok, we compile libraries if needed, do some tricks like ``echo ">foo\n ACGT > test.fa" and so on, but we should keep this to a absolute minimum until we get more space and more funding.

One thing you could do without populating the github repo is to move this into the build script, fetch the fasta etc ... not ideal, but better than nothing.